WebDriverAgentLib/Categories/XCUIElement+FBUtilities.m

```diff
@@ -80,6 +83,64 @@ - (XCElementSnapshot *)fb_lastSnapshot
   return self.lastSnapshot;
 }
 
+- (nullable XCElementSnapshot *)fb_snapshotWithAttributes {
+  if (![FBConfiguration shouldLoadSnapshotWithAttributes]) {
+    return nil;
+  }
+  
+  [self resolve];
+  
+  static NSDictionary *defaultParameters;
+  static NSArray *axAttributes;
+  
+  static dispatch_once_t initializeAttributesAndParametersToken;
+  dispatch_once(&initializeAttributesAndParametersToken, ^{
+    defaultParameters = [[XCAXClient_iOS sharedClient] defaultParameters];
+    // Names of the properties to load. There won't be lazy loading for missing properties,
+    // thus missing properties will lead to wrong results
+    NSArray<NSString *> *propertyNames = @[
+                      @"identifier",
+                      @"value",
+                      @"label",
+                      @"frame",
```

Comment:
does it make sense to include visibleFrame as well?

Reply:
It's not accessed anywhere when we already have the visibility value